src/Symfony/Component/DependencyInjection/Tests/Attribute/AutowireInlineTest.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Tests\Attribute;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\DependencyInjection\Attribute\AutowireInline;
+use Symfony\Component\DependencyInjection\Exception\LogicException;
+use Symfony\Component\DependencyInjection\Reference;
+use function PHPUnit\Framework\assertSame;
```

Comment:
should be removed

Reply:
[Done here](https://github.com/symfony/symfony/compare/4b0e7643ef8444649c452c490fc6335e390d324f..5955b18f44df3374fd1eb423e025f0a0f04d956c)